webapp/channels/src/components/admin_console/system_user_detail/system_user_detail.test.tsx

```diff
@@ -133,6 +135,203 @@ describe('SystemUserDetail', () => {
 
         expect(container).toMatchSnapshot();
     });
+
+    describe('change detection', () => {
+        test('should detect email changes and enable save', async () => {
+            const setNavigationBlockedMock = jest.fn();
+            const props = {
+                ...defaultProps,
+                setNavigationBlocked: setNavigationBlockedMock,
+                intl: {
+                    formatMessage: jest.fn().mockImplementation(({defaultMessage}) => defaultMessage),
+                } as MockIntl,
+            };
+            renderWithContext(<SystemUserDetail {...props}/>);
+
+            await waitFor(() => {
+                expect(screen.queryByTestId('loadingSpinner')).not.toBeInTheDocument();
+            });
+
+            const emailInputs = screen.getAllByDisplayValue(user.email);
+            const emailInput = emailInputs.find((input) => !(input as HTMLInputElement).disabled);
```

Comment:
Could we just select this by its placeholder or label? I don't really understand why we're looking for multiple elements to find one that isn't disabled. If we change that, we can probably get rid of the null check as well since that looks like it could lead to us skipping the assertion if this fails to find the input